spec/rails_helper.rb

```diff
@@ -0,0 +1,64 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+require 'spec_helper'
+ENV['RAILS_ENV'] ||= 'test'
```

Comment:
docker-composeでRAILS_ENV＝development指定してると、developmentのDB見るので下記の方が良い。

```suggestion
ENV['RAILS_ENV'] = 'test'
```